### ISSUE TITLE
Download PDF of HACKTRICKS

### ISSUE THREAD

**Issue by carlospolop** [HackTricks-wiki/hacktricks#66]
Hi there!

There are 2 ways you can obtain a PDF version of HACKTRICKS:
- As a **way to support the development of HACKTRICKS, buy the PDF copy** here: https://github.com/sponsors/carlospolop
- **Share some cool hacking tricks** via push request here. Once your(s) push request(s) are accepted, ask for your PDF copy of hacktricks inside the Telegram group https://t.me/peass. (*Note that push requests fixing syntax/grammar errors are also rewardable, but you should fix a lot of errors, not just one or a few*).

I will try to generate a new version every month. Note that you can only request the PDF of the month when you sent the push request (or of the following month of you want to wait until your push request appears in the PDF).

**IMPORTANT**: The PDF is automatically generated from the gitbook online book. Therefore do not expect it to be perfectly formatted. If you have any suggestions about how to improve the exported PDF share them in this issue. Currently this is being used: https://www.forgebox.io/view/gitbook-exporter

**Comment by carlospolop** [comment]
There is no PDF available anymore, you can run hacktricks locally, just check the main HackTricks and Cloud Hacktricks pages

**Comment by EugeneRaven** [comment]
> There is no PDF available anymore, you can run hacktricks locally, just check the main HackTricks and Cloud Hacktricks pages

Is this local Docker version able to be turned into a PDF/EPUB? Because this local version still wouldn't work on an e-reader.

**Comment by carlospolop** [comment]
No, it won't @EugeneRaven, we no longer support providing a PDF or epub of hacktricks, they were pretty ugly anyways.
